wordpress-mobile/WordPress-iOS
# Aztec editor/integrate latest version of editor

Fixes #https://github.com/wordpress-mobile/WordPress-Aztec-iOS/issues/13

To test:
 - Activate the native editor
 - Create or edit a post/page and see the new editor in action
 - If you are editing you should see the current content of the post
 - No saves/changes of the content are being made, but drafts are created in the blog for new post/pages so be aware of this when using live/critical blogs!

Needs review: @diegoreymendez
